### ISSUE TITLE
unicode error in `_format_exception`

### ISSUE THREAD

**Issue by dpinte** [sjagoe/haas#132]
```
[...]ts\xxx\.devenv\lib\site-packages\haas\result.py", line 198, in inner
    return method(self, *args, **kw)
  File "C:\projects\xxx\.devenv\lib\site-packages\haas\result.py", line 377, in addError
    test, TestCompletionStatus.error, exception=exception)
  File "C:\projects\xxx\.devenv\lib\site-packages\haas\result.py", line 359, in _handle_result
    stderr=stderr,
  File "C:\projects\xxx\.devenv\lib\site-packages\haas\result.py", line 148, in from_test_case
    exception, is_failure, stdout, stderr)
  File "C:\projects\xxx\.devenv\lib\site-packages\haas\result.py", line 84, in _format_exception
    return ''.join(msgLines)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 158: ordinal not in range(128)
[````

**Comment by sjagoe** [comment]
I am going to fix this initially assuming UTF-8, handling the case where UTF-8 may be incorrect by using unicode-replacement. I'm opening a new issue to find a better solution.

**Comment by sjagoe** [comment]
Actually, I can use `locale.getpreferredencoding()`.

**Comment by sjagoe** [comment]
Closed by #133
